engo_glfw_test.go

```diff
@@ -0,0 +1,29 @@
+//+build !netgo,!android
+
+package engo
+
+import (
+	"testing"
+
+	"github.com/github.com/stretchr/testify/assert"
+)
+
+func TestGameSize(t *testing.T) {
```

Comment:
These tests considering that the functions can only return this value

https://github.com/EngoEngine/engo/blob/master/engo_glfw.go#L267-L269